peps/pep-0736.rst

```diff
@@ -48,32 +47,31 @@ and visual noise.
 We contend that a simple syntactic sugar used to simplify this common pattern
 which would confer numerous benefits:
 
-Encourages use of named variables
+Encourages use of named arguments
 ---------------------------------
 
-This syntax would encourage the use of named variables, thereby increasing
-readability (*explicit is better than implicit*) and reducing bugs from argument
-transposition.
+This syntax would encourage the use of named arguments, thereby increasing
+readability and reducing bugs from argument transposition.
```

Comment:
Thanks for your feedback.

I wanted to avoid delaying opening the new PR any longer (to avoid people commenting on old darfts) so I've created it. But I'll have a think about how to make this clearer. Overall though I think:

>how does it increase readability, and how will it reduce bugs

These are both benefits of named arguments, described under 'Rationale'. I didn't want to repeat this only a few sentences later.

> How does it encourage the use of named arguments

That's fair. I think probably goes back to 'The status quo in Python encourages developers (e.g., me, at a minimum) to use shorter and less descriptive names to save keystrokes or use positional arguments to reduce visual clutter.' (quote from [the thread](https://discuss.python.org/t/pep-736-shorthand-syntax-for-keyword-arguments-at-invocation/43432), not in the PEP, but could be added). It's hard to measure but I also haven't received much pushback on this particular point suggesting it's not just me.

Thoughts?